### ISSUE TITLE
Errors in createOnDragStart and createOnDrop?

### ISSUE THREAD

**Issue by stevengoldberg** [erikras/redux-form#1088]
I've been seeing rare errors popping up in my logs from within redux-form and I'm wondering if anyone else has seen these or can offer any insight. I have not been able to repro these myself.

There's this one from `createOnDrop`:
```js
Error: Invalid argument. 
~/redux-form/lib/events/createOnDrop.js?fe65 in Anonymous function at line 9:0

7.  var createOnDrop = function createOnDrop(name, change) {
8.   return function (event) {
9.     change(name, event.dataTransfer.getData(_createOnDragStart.dataKey));
10.  };
11. };
```

And this one from `createOnDragStart`:
```js
Error: Unexpected call to method or property access.
~/redux-form/lib/events/createOnDragStart.js?eaa1 in Anonymous function at line 7:0

4. var dataKey = exports.dataKey = 'value';
5. var createOnDragStart = function createOnDragStart(name, getValue) {
6.  return function (event) {
7.    event.dataTransfer.setData(dataKey, getValue());
8.  };
9. };
```

These occurred with v. 5.2.3, and always with IE11.

**Comment by ooflorent** [comment]
In a few cases `dataTransfer` might not be available. Accesses to `getData` or `setData` would then raise and error.

**Comment by stevengoldberg** [comment]
I looked into this a bit more and [this seems a likely answer to me](http://stackoverflow.com/questions/26213011/html5-dragdrop-issue-in-internet-explorer-datatransfer-property-access-not-pos).

>getData() and setData() attribute must be called exactly "text"

The first param to `setData` and `getData` specifies format, and `redux-form` is passing `"value"`.

**Comment by erikras** [comment]
Fix published in [`v6.0.0-rc.4`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.4).

**Comment by prevostc** [comment]
Still have this issue on v6.4.2

When I initiate a drag on an input I get a createOnDragStart.js:9 Uncaught TypeError: Cannot read property 'setData' of undefined(…)

I don't know what might help to debug this.
